test/integration/ae_mdw/db/sync/contract_test.exs

```diff
@@ -0,0 +1,74 @@
+defmodule AeMdw.Db.Sync.ContractTest do
+  use ExUnit.Case, async: false
+
+  @moduletag :integration
+
+  alias AeMdw.Contract
+  alias AeMdw.Node.Db, as: NodeDb
+  alias AeMdw.Db.Model
+  alias AeMdw.Db.NameTransferMutation
+  alias AeMdw.Db.NameUpdateMutation
+  alias AeMdw.Db.Sync
+
+  require Model
+
+  describe "events_mutations/4" do
+    test "creates name transfer mutation" do
+      {"AENS.transfer", call_txi, _local_idx} =
```

Comment:
currently mainnet/integration has invocations of AENS.transfer and AENS.update internal calls